### ISSUE TITLE
curl bug?

### ISSUE THREAD

**Issue by seandenigris** [svenvc/zinc#53]
```smalltalk
ZnClient new
	url: 'https://myurl.com' asUrl;
	formAt: 'pass' put: 'pa$$w0rd';
	method: #POST;
	curl
```
-> `Error: Improper store into indexable object`

Pharo 9. I delved into the call chain a bit but didn't come up with anything definitive...

**Comment by svenvc** [comment]
That is probably related to the current version of Zn not working 100% on Pharo 9. I hope to fix that this week.

**Comment by svenvc** [comment]
Could you try again with the latest HEAD ? Zn now works fine.

**Comment by seandenigris** [comment]
I confirm it works now. Thanks! Fixed by 205dcfc
